Config/macOS.xcconfig

```diff
@@ -4,4 +4,6 @@ ARCHS = x86_64
 VALID_ARCHS = $(ARCHS) i386
 SKIP_INSTALL = YES
 
+TARGETED_DEVICE_FAMILY = 1, 2
```

Comment:
On macOS config? Isn't 1 for iPhones and 2 for iPads? It's probably wrong right?

Reply:
What's the value for macOS then?

Reply:
I don't think TARGETED_DEVICE_FAMILY is used for mac build.